testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -41,11 +41,18 @@
     SUPPORTS_CREATE_TABLE,
     SUPPORTS_CREATE_TABLE_WITH_DATA(SUPPORTS_CREATE_TABLE),
 
+    SUPPORTS_COMMENT_ON_TABLE,
+    SUPPORTS_COMMENT_ON_COLUMN,
+
+    SUPPORTS_CREATE_VIEW(false),
+
     SUPPORTS_INSERT,
 
     SUPPORTS_DELETE(false),
     SUPPORTS_ROW_LEVEL_DELETE(SUPPORTS_DELETE),
 
+    SUPPORTS_ARRAY,
```

Comment:
Should this also default to false?

Reply:
didn't want to change it here, as there is a number of connectors that support arrays.
it's not like "this feature exists for one connector only, currently" case (as views, or delete as of now)